docs/src/content/quickstart.md

```diff
@@ -4,4 +4,35 @@ name: Quickstart
 
 # Quickstart Guide
 
-This will lead you to different paths.
+This Quickstart tutorial will guide you to setup the Hydrus server, we will be using docker in this tutorial,
+Therefore you can go through the [Docker container engine](https://docker-curriculum.com/) and its best practices deploying virtualized environments.
+
+## Running hydrus server
+> Make sure you have docker-compose installed in your system
+
+#### Getting the [hydrus repository](https://github.com/HTTP-APIs/hydrus)
```

Comment:
Please maintain a consistent heading hierarchy. If a heading is inside h1, it should be h2. if it is inside h2, it should be h3, not h4.  Inconsistent hierarchy might break the inpage navigation.